roles/keystone-defaults/defaults/main.yml

```diff
@@ -51,7 +51,7 @@ keystone:
     - keystone-wsgi-public
   jellyroll: False # custom middleware for password compliance
   security_compliance:
-    enabled: true
+    enabled: false
```

Comment:
might be good to add comment about if turning to true need to remove jellyroll filter:passval from api_paste.ini